source/main.Rmd

```diff
@@ -501,6 +504,19 @@ roc.randomForestModel = pROC::roc(testDataCopy$RainTomorrow,
 auc.randomForestModel = pROC::auc(roc.randomForestModel)
 
 randomForestModel
+
+
+ptm <- proc.time()
+randomForestModel_ranger = train(as.formula(paste('RainTomorrow ~', selectedPredictorsPlus)), 
+   data = trainDataCopy, method = "ranger", trControl = ctrl)
+proc.time() - ptm
+randomForestModel_ranger
+pred.randomForestModel_ranger.prob = predict(randomForestModel_ranger, newdata = testDataCopy, type="prob")
+pred.randomForestModel_ranger.raw = predict(randomForestModel_ranger, newdata = testDataCopy )
+confusionMatrix(data = pred.randomForestModel_ranger.raw, testDataCopy$RainTomorrow)
+
+#ranger method takes less time than the rf method for the randomforest
+#if we tune then rf is much faster than ranger
```

Comment:
Which method would u like to keep? If u want to keep both then you would need to output your findings into report. Without the output it is just a waist of computing time. Makes sense?

Reply:
will keep rf method - and need to delete ranger.
Should I make the change
as even though ranger is fast in processing - rf is much better for model prediction for real data.
since we are wrapping this up - we can go with rf and delete ranger
should I cleanup and resubmit?